projects/swimlane/ngx-ui/src/lib/components/progress-spinner/progress-spinner.component.scss

```diff
@@ -77,15 +75,15 @@
           display: block;
           transform: translate(calc(50% - 35px), calc(50% - 36px));
           fill: none;
-          stroke: var(--spinner-color);
+          stroke: $color-blue-500;
           animation: icon-progress-spinner--check 1s linear forwards;
         }
 
         .spinner-svg--thumbs-down-icon {
           display: block;
           transform: translate(calc(50% - 35px), calc(50% - 28px));
           fill: none;
-          stroke: var(--spinner-color);
+          stroke: $color-red-500;
```

Comment:
Shouldn't these still use spinnerColor?

Reply:
I changed it to match the filter color color blend. These are for the default icons (thumbs up and down) so if you pass in an alternative spinnerColor, the colors will not match.